test/suite/test_sweep04.py

```diff
@@ -0,0 +1,225 @@
+#
+
+import time
+from suite_random import suite_random
+from wiredtiger import stat
+import wttest
+
+# Given a set of values corresponding to successive times,
+# we have an implied set of points in two dimensions.
+# Compute the average for the values and the slope for the
+# least squares regression of the line.
+#
+# We'd use numpy, but we aren't assured that it is always installed
+# for our python.
+def average_slope(y):
+    n = len(y)
+    if n == 0:
+        return [0, 0]  # there's no average or slope
+    elif n == 1:
+        return [y[0], 0]  # there's no slope
+
+    average = sum(y) / n
+    x = range(1, n + 1)  # the implied x axis, steadily increasing integers.
+
+    # Here's a formula for least squares slope:
+    #   https://www.mathsisfun.com/data/least-squares-regression.html
+    top = n * sum(x[i] * y[i] for i in range(n)) - sum(x) * sum(y)
+    bottom = n * sum(x[i]**2 for i in range(n)) - sum(x)**2
+    slope = top / bottom
+    return [average, slope]
+
+@wttest.longtest("lots of files")
+class test_sweep04(wttest.WiredTigerTestCase):
+    tablebase = 'test_sweep04'
+    uri = 'table:' + tablebase
+
+    # Configuration values for the run. If any of these values are changed,
+    # the simulation will run, but the acceptance criteria may fail at the end.
+    core_tables = 10               # Number of core tables that always exist
+    transient_tables = 100         # Number of transient tables at any time
+    transient_examined = 10        # Number of transient tables opened at a time
+    ratio_examined = 0.01          # The chance that transient tables are examined
+    transient_table_max = 10000    # defines the length of the run
+    numkv = 1                      # Number of k/v pairs. Shouldn't matter for this test.
+    nsessions = 100                # Number of sessions in our pool.
+
+    conn_config = 'file_manager=(close_handle_minimum=0,' + \
+                  'close_idle_time=3,close_scan_interval=1),' + \
+                  'statistics=(fast),operation_tracking=(enabled=false),'
+
+    create_params = 'key_format=i,value_format=i'
+
+    def core_uri(self, i):
+        return '%s-c.%d' % (self.uri, i)
+
+    def transient_uri(self, i):
+        return '%s-t.%d' % (self.uri, i)
+
+    def create_table(self, uri):
+        self.session.create(uri, self.create_params)
+        c = self.session.open_cursor(uri, None)
+        for k in range(self.numkv):
+            c[k+1] = 1
+        c.close()
+
+    def examine(self, session, uri_maker, start, count):
+        for i in range(0, count):
+            c = session.open_cursor(uri_maker(start + i))
+            self.assertEquals(c[1], 1)
+            c.close()
+
+    def test_big_run(self):
+        # populate
+        r = suite_random()
+        for i in range(0, self.core_tables):
+            self.create_table(self.core_uri(i))
+
+        created_transient = 0    # running total of transients created
```

Comment:
Take a pass through all the comments to start with capitals and end with periods.

Reply:
Can you do this for all the one-line comments that are complete sentences?